Code/GraphMol/RascalMCES/RascalMCES.cpp

```diff
@@ -265,22 +265,14 @@ void getBondLabels(const ROMol &mol, const RascalOptions &opts,
     } else {
       bondType = std::to_string(b->getBondType());
     }
-    if (b->getBeginAtom()->getAtomicNum() == b->getEndAtom()->getAtomicNum()) {
-      if (b->getEndAtom()->getDegree() < b->getBeginAtom()->getDegree()) {
-        bondLabels[b->getIdx()] = atomLabels[b->getEndAtomIdx()] + bondType +
-                                  atomLabels[b->getBeginAtomIdx()];
-      } else {
-        bondLabels[b->getIdx()] = atomLabels[b->getBeginAtomIdx()] + bondType +
-                                  atomLabels[b->getEndAtomIdx()];
-      }
+    // The atom labels need to be in consistent order irrespective
```

Comment:
Seems like this would be clearer as:
```
    auto lbl1 = atomLabels[b->getBeginAtomIdx()];
    auto lbl2 = atomLabels[b->getEndAtomIdx()];
    if(lbl1 < lbl2) {
      std::swap(lbl1, lbl2);
    }
    bondLabels[b->getIdx()] = lbl1 + bondType + lbl2;
```

Reply:
I assume the swap is very cheap.  In which case the intent is clearer, so yes.  And I guess when the next stage is exponential in N, the set-up time is largely immaterial.